Marlin/src/lcd/language/language_de.h

```diff
@@ -151,65 +154,66 @@
+#define MSG_NO_MOVE                         _UxGT("Motoren einschalten")
 #define MSG_KILLED                          _UxGT("ABGEBROCHEN")
 #define MSG_STOPPED                         _UxGT("ANGEHALTEN")
-#define MSG_CONTROL_RETRACT                 _UxGT("Retract mm")
-#define MSG_CONTROL_RETRACT_SWAP            _UxGT("Wechs. Retract mm")
-#define MSG_CONTROL_RETRACTF                _UxGT("Retract  V")
-#define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Z-Hop mm")
+#define MSG_CONTROL_RETRACT                 _UxGT("Einzug mm")
+#define MSG_CONTROL_RETRACT_SWAP            _UxGT("Wechs. Einzug mm")
+#define MSG_CONTROL_RETRACTF                _UxGT("Einzug   V")
+#define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Z-Sprung mm")
 #define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("Wechs. UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVERF        _UxGT("UnRet  V")
 #define MSG_CONTROL_RETRACT_RECOVER_SWAPF   _UxGT("S UnRet V")
-#define MSG_AUTORETRACT                     _UxGT("Autom. Retract")
+#define MSG_AUTORETRACT                     _UxGT("Autom. Einzug")
 #define MSG_FILAMENTCHANGE                  _UxGT("Filament wechseln")
 #define MSG_FILAMENTLOAD                    _UxGT("Filament laden")
 #define MSG_FILAMENTUNLOAD                  _UxGT("Filament entladen")
 #define MSG_FILAMENTUNLOAD_ALL              _UxGT("Alles entladen")
 #define MSG_INIT_SDCARD                     _UxGT("SD-Karte erkennen")  // Manually initialize the SD-card via user interface
 #define MSG_CNG_SDCARD                      _UxGT("SD-Karte getauscht") // SD-card changed by user. For machines with no autocarddetect. Both send "M21"
-#define MSG_ZPROBE_OUT                      _UxGT("Sensor ausserhalb")
-#define MSG_SKEW_FACTOR                     _UxGT("Skew Faktor")
+#define MSG_ZPROBE_OUT                      _UxGT("Z-Sonde ausserhalb")
+#define MSG_SKEW_FACTOR                     _UxGT("Korrektur Faktor")
 #define MSG_BLTOUCH                         _UxGT("BLTouch")
 #define MSG_BLTOUCH_SELFTEST                _UxGT("BLTouch Test")
-#define MSG_BLTOUCH_RESET                   _UxGT("BLTouch Reset")
+#define MSG_BLTOUCH_RESET                   _UxGT("BLTouch zurücks.")
 #define MSG_BLTOUCH_DEPLOY                  _UxGT("BLTouch ausfahren")
 #define MSG_BLTOUCH_STOW                    _UxGT("BLTouch einfahren")
-#define MSG_MANUAL_DEPLOY                   _UxGT("Z-Sensor anbringen")
-#define MSG_MANUAL_STOW                     _UxGT("Z-Sensor entfernen")
+#define MSG_MANUAL_DEPLOY                   _UxGT("Z-Sonde anbringen")
+#define MSG_MANUAL_STOW                     _UxGT("Z-Sonde entfernen")
 #define MSG_HOME                            _UxGT("Vorher")  // Used as MSG_HOME " " MSG_X MSG_Y MSG_Z " " MSG_FIRST
 #define MSG_FIRST                           _UxGT("homen")
-#define MSG_ZPROBE_ZOFFSET                  _UxGT("Z Versatz")
+#define MSG_ZPROBE_ZOFFSET                  _UxGT("Sonden Z-Versatz")
```

Comment:
if this is this about more than one single zprobe it's fine; otherwise is should be "Sonde Z-Versatz" or "Z-Versatz Sonde" for a single zprobe. Although "Z Probe Versatz" or "Versatz Z Probe" would IMHO be best option because "Z Probe" is more like a name of a thing in German (not limited for just this line).

Reply:
In deutsch, ist, dass der "Sondenversatz" also Singular als der "Sonden Z-Versatz", es gibt nur einen Z-Versatz pro Sonde 😉

Reply:
dann wäre doch "Sondenversatz Z" besser. "Sonden Z-Versatz" hört sich für mich nach mehreren "Z Sonden" an. Wobei "Z-Probe" für mich die Bezeichnung/Name der Baugruppe ist und man diese nicht zu übersetzen braucht, daher wäre "Versatz Z-Probe" eigentlich besser. Wir deutschen sagen ja auch nicht Klapprechner, sondern Laptop oder Notebook (also ich zumindest) -> https://www.dict.cc/deutsch-englisch/Klapprechner.html

Reply:
Okay, einverstanden "Sondenversatz Z" klingt gut